ConfigDownload.cmake

```diff
@@ -0,0 +1,103 @@
+# Download weights for Yolo v3, v4 and v7
+
+# Yolo v3
+# See https://pjreddie.com/darknet/yolo/
+file(DOWNLOAD
+  https://raw.githubusercontent.com/pjreddie/darknet/master/data/coco.names
+  ${CMAKE_CURRENT_SOURCE_DIR}/config/yolo-v3/coco.names
```

Comment:
Please download these to `${CMAKE_CURRENT_BINARY_DIR}` instead of the source directory, and then please add another install rule that installs to `share/${PROJECT_NAME}/config` directory.

The `params.yaml` shouldn't need any modifications. I'd just like to avoid a build modifying files in the source tree.